src/Storefront/Controller/PayPalController.php

```diff
@@ -201,13 +202,17 @@ public function onHandleError(Request $request, SalesChannelContext $context): R
             $request->getSession()->set(self::PAYMENT_METHOD_FATAL_ERROR, $context->getPaymentMethod()->getId());
         }
 
-        $this->logger->warning('Storefront checkout error', [
-            'error' => $request->request->get('error'),
-            'code' => $code,
-            'fatal' => $fatal,
-            'paymentMethodId' => $context->getPaymentMethod()->getId(),
-            'paymentMethodName' => $context->getPaymentMethod()->getName(),
-        ]);
+        $this->logger->log(
+            $fatal || \in_array($code, ['SWAG_PAYPAL__GENERIC_ERROR', 'SWAG_PAYPAL__NOT_ELIGIBLE'], true) ? Level::Error : Level::Warning,
```

Comment:
please make this line less long. Aren't these errors fatal? Or why should the merchant be notified as an error otherwise?

Reply:
I'm afraid that too many errors will confuse people. Are you absolutely sure, these errors are not thrown due to user error etc.?
Also: "not_eligible" can be due to user, right? e.g. SEPA not available to US IP addresses?